ntbs-service/Models/Entities/Notification.cs

```diff
@@ -107,6 +107,22 @@ public Notification()
             ClinicalDetails.IsPostMortem != true || (TreatmentEvents != null && TreatmentEvents.Any(x =>
                 x.TreatmentEventTypeIsOutcome && x.TreatmentOutcome.TreatmentOutcomeType == TreatmentOutcomeType.Died));
 
+        public bool IsInactive()
+        {
+            var lastTreatmentEvent = TreatmentEvents.OrderByDescending(t => t.EventDate)
+                .ThenBy(t => t.TreatmentEventTypeIsOutcome)
+                .FirstOrDefault();
+            if (lastTreatmentEvent != null)
+            {
+                return lastTreatmentEvent.TreatmentEventTypeIsOutcome &&
+                       lastTreatmentEvent.TreatmentOutcome?.TreatmentOutcomeSubType !=
+                       TreatmentOutcomeSubType.StillOnTreatment &&
+                       lastTreatmentEvent.EventDate < DateTime.Today.AddYears(-1);
```

Comment:
A few things:

- You can make this simpler by including the `lastTreatmentEvent != null` condition in the `return` statement (so you don't need the `if` block at all).
- I think it's clearer if the `&&` operator comes on a new line (I'm surprised our `.editorconfig` doesn't have a rule for this - maybe we should add one?)
- If a comparison is split across multiple lines, then the second line should be indented (in this case, having the whole condition on one line would seem reasonable though)

Reply:
Thanks for picking up those formatting issue, they are fixed now. Unfortunately there doesn't seem to be an `.editorconfig` setting for the position of operators in multi-line expressions, so we won't be able to enforce this automatically

Reply:
That's a shame :-(